components/organisms/Footer/footer.tsx

```diff
@@ -60,7 +60,7 @@ const footerContext = [
 const Footer = (): JSX.Element => {
   return (
     <footer className=" h-24 w-full bg-light-slate-2 transition">
-      <div className=" container mx-auto px-2 md:px-16  lg:border-t lg:py-8 lg:items-center lg:justify-between lg:gap-x-4 flex flex-col gap-y-4 lg:flex-row py-2 w-full">
+      <div className=" container mx-auto px-2 md:px-16  lg:border-t lg:py-8 lg:items-center lg:justify-between lg:gap-x-4 flex flex-col gap-y-4 lg:flex-row py-2 w-full" style="padding-bottom:4rem;">
```

Comment:
We're using Tailwind for the styling of the project. Please use the corresponding class from Tailwind.